website/docs/zh/api/_meta.json

```
@@ -3,15 +3,13 @@
   "cli",
   {
     "type": "dir",
-    "name": "modules",
-    "label": "模块"
+    "name": "runtime-api",
+    "label": "运行时 API"
   },
-  "javascript-api",
-  "hmr",
   {
     "type": "dir",
-    "name": "objects",
-    "label": "对象"
+    "name": "javascript-api",
+    "label": "构建对象 API"
```

I prefer using `JavaScript API` in Chinese doc too

```suggestion
    "label": "JavaScript API"
```